app/views/users/show.html.erb

```diff
@@ -1,3 +1,16 @@
-<h1>User Details</h1>
-<p><%= @placeholder_text %></p>
-<p>Find me in app/views/users/show.html.erb</p>
+<%= render partial: 'users/user', locals: { user: @user } %>
+<section id="user-details">
+  <div>
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+  <hr>
+  <section id="posts-container">
+    <% @user.posts.each do |post| %>
+      <%= render partial: 'posts/post', locals: { post: post } %>
+    <% end %>
+  </section>
```

Comment:
- [ ] Great job working on the views so far but I think it would be great if you made like the given wireframe since you should at least make the same layout so kindly adjust your element so you have the image for the user and his/her name in a box beside the image then a box contains the bio then the posts please make them the same given wireframe 🔥💪🏻🎉


![image](https://github.com/richie1988/blog-app/assets/58553711/ead20f39-8e2f-4e77-b06f-e530d1fe39b7)


- Your
![image](https://github.com/richie1988/blog-app/assets/58553711/024a723a-f90a-4869-a414-afd2dd1b70fa)



- Given

![image](https://github.com/richie1988/blog-app/assets/58553711/07599433-4395-44d5-84a1-3246eb2b828c)